controllers/mlmd_test.go

```diff
@@ -131,8 +131,9 @@ func TestDeployMLMDV2(t *testing.T) {
 	// Construct DSPA Spec with MLMD Enabled
 	dspa := &dspav1alpha1.DataSciencePipelinesApplication{
 		Spec: dspav1alpha1.DSPASpec{
-			DSPVersion: "v2",
-			APIServer:  &dspav1alpha1.APIServer{},
+			DSPVersion:  "v2",
+			PodToPodTLS: boolPtr(false),
```

Comment:
Why can't be the following: If you specify the cert, then this should be true otherwise false.

Reply:
they are mutually exclusive, you specify certs to add to the list of certs we need to recognize
you can opt not to specify custom certs, and podtopodtls and still use podtopodtls